test/e2e-v2/cases/mongodb/docker-compose.yml

```diff
@@ -0,0 +1,109 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '2.1'
+
+services:
+  oap:
+    extends:
+      file: ../../script/docker-compose/base-compose.yml
+      service: oap
+    ports:
+      - "12800:12800"
+    networks:
+      - e2e
+  mongodb-1:
+    image: mongo:6.0.8
+    networks:
+      - e2e
+    command: mongod --replSet rs1 --shardsvr --port 27017
+    healthcheck:
+      test: [ "CMD", "bash", "-c", "cat < /dev/null > /dev/tcp/127.0.0.1/27017" ]
+      interval: 10s
+      timeout: 10s
+      retries: 120
+  mongodb-2:
+    image: mongo:6.0.8
+    networks:
+      - e2e
+    command: mongod --replSet rs1 --shardsvr --port 27017
+    healthcheck:
+      test: [ "CMD", "bash", "-c", "cat < /dev/null > /dev/tcp/127.0.0.1/27017" ]
+      interval: 10s
+      timeout: 10s
+      retries: 120
+  mongodb-repl-setup:
+    image: mongo:6.0.8
+    networks:
+      - e2e
+    depends_on:
+      mongodb-1:
+        condition: service_healthy
+      mongodb-2:
+        condition: service_healthy
+    volumes:
+      - ./setup.sh:/setup.sh
+    entrypoint: [ "bash", "/setup.sh" ]
+  mongodb-exporter-1:
+    image: percona/mongodb_exporter:0.39
```

Comment:
I suggest to use the image packaged by bitnami https://github.com/bitnami/containers/tree/main/bitnami/mongodb-exporter, although `bitnami/mongodb-exporter` is packaged based on `percona/mongodb_exporter`, `bitnami/mongodb-exporter` is used in mongodb official Helm Chart, meaning it makes more sense for us to test the bitnami one

Reply:
Give me some times, I will do some tests on bitnami/mongodb-exporter.

Reply:
This helm chart looks good. https://github.com/prometheus-community/helm-charts/tree/main/charts/prometheus-mongodb-exporter

Reply:
`percona/mongodb_exporter` is an official image.  [Helm Chart](https://github.com/prometheus-community/helm-charts/tree/main/charts/prometheus-mongodb-exporter) used `percona/mongodb_exporter` image. I want to know why you recommend the Bitnami Helm chart. @kezhenxu94

Reply:
I tested `bitnami/mongodb-exporter`, it works too. So please help me determine which one to use @innerpeacez @kezhenxu94

Reply:
Forgive me, I meant the official MongoDB Helm Chart, not the official exporter Helm Chart, I mentioned this because I have a running MongoDB replica set and found the exporter is bitnami's, but I just realize the running MongoDB is not deployed with the official one https://github.com/mongodb/helm-charts/tree/main/charts, instead, it's deployed with the bitnami's too. Then I'm ok to keep it as is in this PR